### PR TITLE
Task task10061 create ui feature option sceen

### PR DESCRIPTION
**Pull request by HaLanHuynh**
Title: Create UI (Feature Option Screen)

Link: https://edu-redmine.sun-asterisk.vn/issues/10061

Sceen:
57730372_447649229303163_5608492277937733632_n

### CODE REVIEW

**Comment by HaLanHuynh** [comment]
ready

**Comment by HaLanHuynh** [comment]
ready

**Comment by tuanlvt-1117** [comment]
commented

**Comment by HaLanHuynh** [comment]
ready

**Comment by HaLanHuynh** [comment]
ready

File: app/src/main/res/layout/fragment_home.xml [hunk 1]
```diff
@@ -205,10 +205,62 @@
...
+            android:layout_width="wrap_content"
+            android:layout_height="@dimen/dp_40"
+            android:layout_marginTop="@dimen/dp_130"
+            android:layout_marginLeft="@dimen/dp_25"
+            android:text="@string/temperature"
+            android:textSize="@dimen/sp_20"
+            android:textColor="@color/colorWhite" />
+    </android.support.design.widget.NavigationView>
 </android.support.v4.widget.DrawerLayout>
+
```
Comment (tuanlvt-1117) [inline app/src/main/res/layout/fragment_home.xml]: remove blank

**Comment by HaLanHuynh** [comment]
ready

**Review by tuanlvt-1117 — APPROVED** [review]

**Pull request merged**